src/types/RootState.ts

```diff
@@ -17,7 +17,7 @@ export interface RootState {
   account: AccountState
   network: NetworkState
   transaction: TransactionState
-  ledger: LedgerState
+  multiAccounts: MultiAccountsState
```

Comment:
I'd name it importAccounts or chooseAccountsToImport

and I'd name PR "Derive multiple accounts from mnemonic"